bin/pkg.ml

```diff
@@ -320,8 +320,15 @@ module Lock = struct
         ]
   ;;
 
+  let xdg_repo_location =
+    let ( / ) = Filename.concat in
```

Comment:
This is kind of funny since it will differ on windows but later you use `/`. I think you should just convert the Xdg to a path directly and use `Path.L.relative` which takes in a list of args rather than a single on like `Path.relative`.

Reply:
I guess you didn't introduce this in this PR however, so we can do it later.

Reply:
You're right. Might as well bite the bullet here.